### PR TITLE
adding label and compound label pattern

### PR DESCRIPTION
**Pull request by terezanovotna**
Adding label and compound label pattern from #680 proposal

### CODE REVIEW

**Comment by terezanovotna** [comment]
@mcarrano

File: pattern-library/forms-and-controls/labels/design/overview.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Labels
```
Comment (mcarrano) [inline pattern-library/forms-and-controls/labels/design/overview.md]: There seems to be a problem with the image links on this page as I am not seeing either of the images.

Comment (terezanovotna) [reply pattern-library/forms-and-controls/labels/design/overview.md]: Hi Matt, the images seems to be loaded properly now. 
https://github.com/patternfly/patternfly-design/blob/781ebd867e3b266deee5109987553eecc52e8b0b/pattern-library/forms-and-controls/labels/design/design.md

Comment (terezanovotna) [reply pattern-library/forms-and-controls/labels/design/overview.md]: In terms of examples, I used to have a paragraph about it in the proposal, but then we all agreed it is not necessary. What are you suggestions?

https://docs.google.com/document/d/19Vu9Q_lfCQDkwO3LQuwsImSH0rH9i0o9AF24Uer6WqA/edit?usp=sharing

Comment (mcarrano) [reply pattern-library/forms-and-controls/labels/design/overview.md]: I don't have a good answer here, so I am going to go ahead and merge this. We can revisit later, if needed. Thanks for your work on this @terezanovotna !

**Pull request merged**